docs/sphinx/developers/java-library.txt

```diff
@@ -37,55 +38,152 @@ repositories element can be defined at the top of your inheritance chain.
+    :header-rows: 1
+
+    * - Package
+      - Maven name
+      - License
+    * - `Logback Classic v1.1.1 <http://logback.qos.ch>`_
+      - ch.qos.logback:logback-classic:1.1.1
+      - `Eclipse Public License v1.0`_
+    * - `Logback Core v1.1.1 <http://logback.qos.ch>`_
+      - ch.qos.logback:logback-core:1.1.1
+      - `Eclipse Public License v1.0`_
+    * - `JHDF5 v14.12.0 <https://wiki-bsse.ethz.ch/display/JHDF5>`_
+      - ch.systems.cisd:jhdf5:14.12.0
+      - `Apache License v2.0`_
+    * - `XMP Library for Java v5.1.2 <http://www.adobe.com/devnet/xmp.html>`_
+      - com.adobe.xmp:xmpcore:5.1.2
+      - `BSD License`_
+    * - `JCommander v1.27 <http://beust.com/jcommander>`_
+      - com.beust:jcommander:1.27
+      - `Apache License v2.0`_
+    * - `metadata-extractor v2.6.2 <https://github.com/drewnoakes/metadata-extractor>`_
+      - com.drewnoakes:metadata-extractor:2.6.2
+      - `Apache License v2.0`_
+    * - `Kryo v2.24.0 <http://github.com/EsotericSoftware/kryo>`_
+      - com.esotericsoftware.kryo:kryo:2.24.0
+      - `BSD License`_
+    * - `MinLog v1.2 <https://github.com/EsotericSoftware/minlog>`_
+      - com.esotericsoftware.minlog:minlog:1.2
+      - `BSD License`_
+    * - `Guava v17.0 <http://github.com/google/guava>`_
+      - com.google.guava:guava:17.0
+      - `Apache License v2.0`_
+    * - `JGoodies Common v1.7.0 <http://www.jgoodies.com/downloads/libraries/>`_
+      - com.jgoodies:jgoodies-common:1.7.0
+      - `BSD License`_
+    * - `JGoodies Forms v1.7.2 <http://www.jgoodies.com/downloads/libraries/>`_
+      - com.jgoodies:jgoodies-forms:1.7.2
+      - `BSD License`_
+    * - `Commons Collections v3.2.1 <http://commons.apache.org/collections/>`_
+      - commons-collections:commons-collections:3.2.1
+      - `Apache License v2.0`_
+    * - `Commons Lang v2.4 <http://commons.apache.org/lang/>`_
+      - commons-lang:commons-lang:2.4
+      - `Apache License v2.0`_
+    * - `Commons Logging v1.1.1 <http://commons.apache.org/logging/>`_
+      - commons-logging:commons-logging:1.1.1
+      - `Apache License v2.0`_
+    * - `NetCDF-Java Library v4.3.19 <http://www.unidata.ucar.edu/software/netcdf-java/documentation.htm>`_
+      - edu.ucar:netcdf:4.3.19
+      - `MIT-Style License`_
+    * - `Joda time v2.2 <http://github.com/JodaOrg/joda-time>`_
+      - joda-time:joda-time:2.2
+      - `Apache License v2.0`_
+    * - `JUnit v4.10 <http://www.junit.org>`_
+      - junit:junit:4.10
+      - `Common Public License v1.0`_
+    * - `Apache Log4j v1.2.17 <http://logging.apache.org/log4j/1.2>`_
+      - log4j:log4j:1.2.17
+      - `Apache License v2.0`_
+    * - `ImageJ v1.48s <http://imagej.net>`_
+      - net.imagej:ij:1.48s
+      - Public domain
+    * - `Assume NG v1.2.4 <http://github.com/hierynomus/assumeng>`_
+      - nl.javadude.assumeng:assumeng:1.2.4
+      - `Apache License v2.0`_
+    * - `Apache Velocity v1.6.4 <http://velocity.apache.org/engine/releases/velocity-1.6.2>`_
```

Comment:
Use http://velocity.apache.org/ since the version-specific link has been retired